File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/markUnused/IMarkUnused.java
package org.ngbp.jsonrpc4jtestharness.rpc.markUnused;


import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod;
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType;


@JsonRpcType
public interface IMarkUnused {

    @JsonRpcMethod("org.atsc.cache.markUnused")
    Object markUnused();
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/markUnused/IMarkUnused.kt
package org.ngbp.jsonrpc4jtestharness.rpc.markUnused

import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcMethod
import com.github.nmuzhichin.jsonrpc.annotation.JsonRpcType
import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

@JsonRpcType
interface IMarkUnused {
    @JsonRpcMethod("org.atsc.cache.markUnused")
    fun markUnused(): RpcResponse
}